backend/apis/utils/transform-utils.js

```diff
@@ -1,7 +1,9 @@
 exports.responseDateWithoutTime = date => {
-  const isoString = new Date(date).toISOString();
-  return isoString.slice(0, isoString.indexOf("T"));
+  const dateFormat = require("dateformat");
```

Comment:
We should not be doing `require` in the middle of the function (which is CommonJS), but `import` at the top of this file (which is ESM). See https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/beeeb2098431d826e3292198a66624a2c03eac15/frontend/client-list/table/mobile-clients-table.component.tsx#L4 for an example of what I mean.

Reply:
since this is on the backend should a change still be made?

Reply:
Nope good call -- backend uses require(), not import. The normal thing to do is to add all require() calls to the top of the file, but calling require here works just as well. I'll approve and merge.